user_tools/src/spark_rapids_pytools/rapids/qualification.py

```diff
@@ -551,20 +571,18 @@ def __build_global_report_summary(self,
         target_platform = self.ctxt.get_ctxt('targetPlatform')
         if target_platform is not None:
             pricing_config = self.ctxt.platform.configs.get_value_silent('csp_pricing')
+        if pricing_config is None:
+            # OnPrem platform doesn't have pricing information. We do not calculate cost savings for
+            # OnPrem platform if the target_platform is not specified.
+            self.logger.warning('The pricing configuration for the given platform is not defined.\n\t'
+                                'Savings estimates cannot be generated.')
```

Comment:
Nit: in line 234, the logger uses "Savings calculations" and here it uses "Savings estimates", not sure if it is better to make the terms consistent.